system_test/aest_sync_SUITE.erl

```diff
@@ -201,8 +204,8 @@ new_node_joins_network(Cfg) ->
     StartTime = os:timestamp(),
 
 
-    inject_spend_txs(old_node1, patron(), 20, 1, 100),
-    inject_spend_txs(old_node2, patron(), 20, 21, 100),
+    inject_spend_txs(old_node1, patron(), 5, 1, 100),
```

Comment:
I see these reductions in number of spend txs are independent change for making test faster. Ok.

Reply:
Not just making them faster, also to make sure we don't spend too much time posting, giving the chain the possibility to grow over the target height.